modules/object-collections.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * operating/create-use-collections.adoc
+:_content-type: CONCEPT
+[id="understanding-object-collections_{context}"]
+= Understanding object collections
+
+You can create an object collection that can be used across namespaces, clusters, and deployments. Some common use cases for collections include:
+
+* Creating a group of namespaces to identify an infrastructure area that is owned by a specific development team
+* Using a deployment label to group all of the deployments in a distributed application spanning multiple namespaces
+* Identifying an application used in used in multiple clusters, such as a development and production cluster, which requires the same policy regardless of location
+* Performing cluster mapping for an entire production or test environment
+* Using annotations to identify the owner of a vulnerability issue that impacts a range of deployments or images
```

Comment:
Annotations are not supported for this initial MVP release, at least in the UI. Collections can only be defined by rules that target cluster/namespace/deployment names or labels.

I know there was discussion early on about supporting collection rules targeting annotations, but I'm not sure where we landed on the timeline for that. Boaz or Mandar would probably have more accurate info.

Reply:
I have the same understanding as Dave